cumulusci/tasks/bulkdata/tests/test_delete.py

```diff
@@ -10,12 +13,15 @@
     DataOperationType,
 )
 from cumulusci.tasks.bulkdata.tests.utils import _make_task
+from cumulusci.tasks.bulkdata.tests.test_utils import mock_describe_calls
 
 
 class TestDeleteData(unittest.TestCase):
+    @responses.activate
     @mock.patch("cumulusci.tasks.bulkdata.delete.BulkApiQueryOperation")
     @mock.patch("cumulusci.tasks.bulkdata.delete.BulkApiDmlOperation")
     def test_run(self, dml_mock, query_mock):
+        mock_describe_calls()
```

Comment:
Doesn't have to happen now (it requires refactoring to not use unittest.TestCase) but this looks like a good candidate for a class-scoped pytest fixture